tenant-cd-api/src/main/java/ai/vespa/hosted/cd/internal/TestRuntimeProvider.java

```diff
@@ -0,0 +1,24 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package ai.vespa.hosted.cd.internal;
+
+import ai.vespa.hosted.cd.TestRuntime;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * @author mortent
+ */
+public interface TestRuntimeProvider  {
+
+    static final AtomicReference<TestRuntime> testRuntime = new AtomicReference<>();
```

Comment:
A `public static final` field that is a mutable object is a bit awkward
 for an interface. Should we make this an abstract class instead?